flyteidl/protos/flyteidl/service/admin.proto

```diff
@@ -504,6 +504,19 @@ service AdminService {
     };
   }
 
+  // Fetches a :ref:`ref_flyteidl.admin.Project`
+  rpc GetProject (flyteidl.admin.Project) returns (flyteidl.admin.Project) {
```

Comment:
kind of odd that the request and response message types are the same. Is it worthwhile defining a ProjectGetRequest, like so?

```
message ProjectGetRequest {
    // Indicates a unique project.
    // +required
    string id = 1;
}
```